leaf-ui/rappid-extensions/ElementInspector.js

```diff
@@ -219,38 +225,48 @@ var ElementInspector = Backbone.View.extend({
         }
 
         this.updateCell();
-        */
+        
+    },  
+    
+    /**
+    * Removes the view so we don't have multiple ones in the sidebar
+    */
+     removeView: function(){
+        this.remove();
     },
 
     /**
      * Checks the initial satisfaction value for a (no value).
      * If the initial satisfaction value is (no value), then set the
-     * availible function options to be No Function, Stochastic and UserDefined
+     * availible function options to be Stochastic
      * If not, set the function options so that all options are availible
-     */
-    checkInitialSatValue() {
-        if (this.intention.getInitialSatValue() == '(no value)') {
-            // Stochastic should be the only option for '(no value)'
-            this.$('.function-type').empty();
-            this.$('.function-type').append('<option value=R> Stochastic </option>');
-        } else {
-            this.$('.function-type').empty();
-            this.$('.function-type').append('<option value=NT> No Function </option>');
-            this.$('.function-type').append('<option value=C> Constant </option>');
-            this.$('.function-type').append('<option value=R> Stochastic </option>');
-            this.$('.function-type').append('<option value=I> Increase </option>');
-            this.$('.function-type').append('<option value=D> Decrease </option>');
-            this.$('.function-type').append('<option value=RC> Stochastic-Constant </option>');
-            this.$('.function-type').append('<option value=CR> Constant-Stochastic </option>');
-            this.$('.function-type').append('<option value=MP> Montonic Positive </option>');
-            this.$('.function-type').append('<option value=MN> Montonic Negative </option>');
-            this.$('.function-type').append('<option value=SD> Satisfied Denied </option>');
-            this.$('.function-type').append('<option value=DS> Denied Satisfied </option>');
-            this.$('.function-type').append('<option value=UD> User Defined </option>');
+    */
+  checkInitialSatValue: function() {
+
+          if (this.model.get('intention').getInitialSatValue() == '(no value)') {
```

Comment:
Thoughts on using a class that can hide or show options based on if it is Stochastic or not? Check out what was done in the LinkInspector.js and ask @Domily11 if you have questions because she did something similar there and it could clean up this function a lot

Reply:
Nvm we got it!!!